source/features/index.tsx

```diff
@@ -59,11 +59,15 @@ const logError = (url: string, error: unknown): void => {
 	// Don't change this to `throw Error` because Firefox doesn't show extensions' errors in the console
 	console.group('❌', id, version, pageDetect.isEnterprise() ? 'GHE →' : '→', error);
 
+	const searchIssueUrl = new URL('https://github.com/refined-github/refined-github/issues');
+	searchIssueUrl.searchParams.set('q', `is:issue "${message}"`);
```

Comment:
Not sure if quoting them is better, at least that's how it looks to me.

Reply:
Maybe not, issues may still be useful even if they're not identical

Reply:
> Maybe not, issues may still be useful even if they're not identical

Somehow I missed see this message 😅